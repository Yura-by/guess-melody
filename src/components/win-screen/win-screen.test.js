import React from 'react';
import renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import WinScreen from './win-screen.jsx';
import history from '../../history.js';

it(`WinScreen snapshot test`, () => {
  const tree = renderer.create(<Router history={history}>
    <WinScreen
      gameTime={12}
      currentTime={9}
      mistakes={3}
      onResetGame={jest.fn()}
    />
  </Router>).toJSON();

  expect(tree).toMatchSnapshot();
});
