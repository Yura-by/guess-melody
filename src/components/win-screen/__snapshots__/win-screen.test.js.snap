// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WinScreen snapshot test 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="result__total"
  >
    За 0 минуты и 3 секунд вы ответили на все вопросы, совершив 3 ошибки
  </p>
  <a
    className="replay"
    href="/"
    onClick={[Function]}
  >
    Сыграть ещё раз
  </a>
</section>
`;
